horovod/spark/lightning/datamodule.py

```diff
@@ -99,7 +101,7 @@ def train_dataloader(self):
             # To avoid loading too much data in memory, need to calculate the queue size
             # dynamicaly, and limit the data loaded in queue.
             # Add 1 in size for storing the None in the end of each epoch.
-            kwargs['async_loader_queue_size'] = max(1, min(100000 // kwargs['batch_size'], kwargs['limit_step_per_epoch'] // 4)) + 1
+            kwargs['async_loader_queue_size'] = max(1, min(2000000 // kwargs['batch_size'], kwargs['limit_step_per_epoch'] // 4)) + 1
```

Comment:
How is this 100000 or 2000000 calculated based on? This looks like a significant increase. If this is not dynamically determined, maybe we can define some constants.

Reply:
It estimated based on deepETA's data. Only an estimation because the real data size in MB is unknown here.

Reply:
hmm ok, seems arbitrary. if possible, would be good to pass this in as an argument from our side so that we dont need to modify the horovod part again if anything changes.

Reply:
Modify the code to let user pass the queue size.